File: MS/src/main/java/com/yskc/ms/entity/ms/QualityScoreLog.java
package com.yskc.ms.entity.ms;

import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;
import com.fasterxml.jackson.annotation.JsonFormat;
import com.yskc.ms.entity.ms.models.MsBaseEntity;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.Date;

/**
 * @DateTime: 2022/6/10 10:29
 * @Description:评分记录日志
 * @author: hsx
 */
@TableName("quality_score_log")
public class QualityScoreLog extends MsBaseEntity implements Serializable {

    private static final long serialVersionUID = 6199976500544795804L;

    @TableId
    private Integer id;

    private Integer qualityId;

    private Integer scoreCount;

    private String scores;

    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private BigDecimal totalScore;

    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private BigDecimal avgScore;

    private Date submitTime;

    private Date auditTime;

    private Integer engineerId;

    private Date month;

    private Boolean passed;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getQualityId() {
        return qualityId;
    }

    public void setQualityId(Integer qualityId) {
        this.qualityId = qualityId;
    }

    public Integer getScoreCount() {
        return scoreCount;
    }

    public void setScoreCount(Integer scoreCount) {
        this.scoreCount = scoreCount;
    }

    public String getScores() {
        return scores;
    }

    public void setScores(String scores) {
        this.scores = scores;
    }

    public BigDecimal getTotalScore() {
        return totalScore;
    }

    public void setTotalScore(BigDecimal totalScore) {
        this.totalScore = totalScore;
    }

    public BigDecimal getAvgScore() {
        return avgScore;
    }

    public void setAvgScore(BigDecimal avgScore) {
        this.avgScore = avgScore;
    }

    public Date getSubmitTime() {
        return submitTime;
    }

    public void setSubmitTime(Date submitTime) {
        this.submitTime = submitTime;
    }

    public Date getAuditTime() {
        return auditTime;
    }

    public void setAuditTime(Date auditTime) {
        this.auditTime = auditTime;
    }

    public Integer getEngineerId() {
        return engineerId;
    }

    public void setEngineerId(Integer engineerId) {
        this.engineerId = engineerId;
    }

    public Date getMonth() {
        return month;
    }

    public void setMonth(Date month) {
        this.month = month;
    }

    public Boolean getPassed() {
        return passed;
    }

    public void setPassed(Boolean passed) {
        this.passed = passed;
    }
}

File: YSDB/MSDB/Models/quality_score_log.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MSDB.Models
{
    public class quality_score_log : tablebase
    {
        [Key]
        public int id { get; set; }

        public int qualityId { get; set; } 

        ///<summary>
        ///打分次数 超过三次（包含第三次），所有得分为0
        ///</summary>
        public int scoreCount { get; set; }

        ///<summary>
        /// 得分详情
        ///</summary>
        [MaxLength(500)]
        public string scores { get; set; }

        ///<summary>
        /// 当次总分
        ///</summary>
        [Column(TypeName = "decimal(5,2)")]
        public decimal totalScore { get; set; }

        ///<summary>
        /// 平均分(当scoreCount大于等于3时，该值为0)
        ///</summary>
        [Column(TypeName = "decimal(5,2)")]
        public decimal avgScore { get; set; }

        ///<summary>
        ///提交时间
        ///</summary>
        public DateTime submitTime { get; set; }

        ///<summary>
        ///审核时间
        ///</summary>
        public DateTime auditTime { get; set; }
        
        ///<summary>
        /// 工程师ID
        ///</summary>
        public int engineerId { get; set; }
        ///<summary>
        ///评分月份
        ///</summary>
        public DateTime month { get; set; }
        ///<summary>
        ///通过
        ///</summary>
        public bool passed { get; set; }
    }
}
